api/spec/requests/spree/api/v2/platform/taxons_spec.rb

```diff
@@ -80,4 +80,68 @@
       end
     end
   end
+
+  describe 'taxons#update for metadata' do
```

Comment:
@aleksandarpetrushev can we also create action?

Reply:
yes

Reply:
will try to refactor out shared examples